doc/deployment/README.md

```diff
@@ -36,10 +44,21 @@ The base image may be used in the standalone mode:
 ```bash
 docker run \
     --network host \
-    emcmongoose/mongoose[-<TYPE>] [\
-    <ARGS>]
+    emcmongoose/mongoose[-<TYPE>] \
+    [<MONGOOSE CLI ARGS>]
 ```
 
+##### Mount files
+
+An example for mounting and using a scenario. Thus, files for input/output, configurations, logs and metrics can be mounted.
```

Comment:
I would strongly encourage adding next info: Mounting a volume is the only right way to save Mongoose logs in docker. Don't get confused by `--output-file` option, it's not about docker. 
Probably this info should be added somewhere else